package.json

```diff
@@ -40,7 +40,8 @@
     "typescript": "^4.9.3"
   },
   "files": [
-    "dist/**/*"
+    "dist/**/*",
+    "dist-cjs/**/*"
```

Comment:
Can we have all distribution files under a singular `dist/` with subdirectories for cjs, es, and coverage?

Reply:
Yes, I can change that in next release since this will be transparent for matter.js users